library/psa_crypto_pake.c

```diff
@@ -404,6 +391,11 @@ static psa_status_t psa_pake_ecjpake_setup( psa_pake_operation_t *operation )
     if( ret != 0 )
         return( mbedtls_ecjpake_to_psa_error( ret ) );
 
+    mbedtls_platform_zeroize( operation->password, operation->password_len );
```

Comment:
I think it would be better to do unconditionally right after `mbedtls_ecjpake_setup()`, that if before the `if( ret != 0 )` that possibly returns.